src/thing_objects.c

```diff
@@ -1750,7 +1749,110 @@ TngUpdateRet object_update_object_scale(struct Thing *objtng)
 
 TngUpdateRet object_update_power_sight(struct Thing *objtng)
 {
-    return _DK_object_update_power_sight(objtng);
+    int result; // eax
+
+    objtng->health = 2;
+
+    static const long effkind[] = {43, 86, 87, 88};
```

Comment:
Would be nice to define this outside the function, like `lightning_spangles`. Naming it `effkind` is not smart, because we have 'effects' and 'effect elements' and this is an effect element'.

You can also use the `enum ThingEffectElements` for these numbers:
    TngEffElm_RedTwinkle,
    TngEffElm_BlueTwinkle,
    TngEffElm_GreenTwinkle,
    TngEffElm_YellowTwinkle,